test/test_helper.exs

```diff
@@ -27,6 +44,15 @@ defmodule TestHelper do
     %{store: store, module: wasm_module}
   end
 
+  def wasm_link_module do
+    {:ok, store} = Wasmex.Store.new()
+
+    {:ok, wasm_module} =
+      Wasmex.Module.compile(store, File.read!(TestHelper.wasm_link_test_file_path()))
+
+    %{store: store, module: wasm_module}
+  end
+
```

Comment:
cool! love to have this helper, now we just need to write actual tests to call into this helper, right?